lib/commands/new.ts

```diff
@@ -1,9 +1,13 @@
 import * as fs from 'fs';
 import { TemplateFileNotFoundError } from '../errors';
+import { Timestamp } from '../utils/timestamp';
 
 interface CommandNewOptions {
   migrationsDir: string;
-  migrationName?: string;
+  migrationName: {
+    title?: string;
+    timestampFormat: string;
+  };
```

Comment:
I think it would be simpler to keep these parameters separate

Reply:
okay 👍. Separated: https://github.com/mycodeself/mongo-migrate-ts/pull/99/commits/181d1e519d3f69bd7608840cc5ad7c35557e24c8